apps/desktop/src/lib/file/FileListItem.svelte

```diff
@@ -45,22 +46,13 @@
 	);
 
 	const selectedFiles = fileIdSelection.files;
-
-	const draggableFiles = $derived.by(() => {
-		if ($selectedFiles?.some((selectedFile) => selectedFile.id === file.id)) {
-			return $selectedFiles || [];
-		} else {
-			return [file];
-		}
-	});
+	const draggable = !readonly && !isUnapplied;
 
 	let contextMenu = $state<ReturnType<typeof FileContextMenu>>();
-
 	let draggableEl: HTMLDivElement | undefined = $state();
-	let checked = $state(false);
+	let draggableFile: DraggableFile | undefined;
```

Comment:
Why does it make sense to declare this here? I don't see it used outside the effect. If it is used outside the effect, should it be a `$state()`?

Reply:
Thanks, that was a mistake, the variable is not needed.